### DISCUSSION TITLE
use wlan1 as internet and wlan0 as AP setting requirements

### DISCUSSION THREAD

**Discussion by riker65** [RaspAP/raspap-webgui#1196] [Q&A]
Hi

I try to set wlan0 as AP using wlan1 as internet access.

Try to find the right setting, what should I do?

also try to tether/bridge wlan1 to eth0

see hardcopy

Thanks for hints

**Comment by riker65** [comment]
looks like can not add hardcopy.

for wlx74da38005318 ping one.one.one.one is working
but not for wlan0

log is:
`default via 10.3.141.1 dev wlan0 src 10.3.141.1 metric 303
default via 192.168.10.1 dev wlx74da38005318 proto dhcp src 192.168.10.115 metric 304
10.3.141.0/24 dev wlan0 proto dhcp scope link src 10.3.141.1 metric 303
192.168.10.0/24 dev wlx74da38005318 proto dhcp scope link src 192.168.10.115 metric 304`


wlan0:
`3: wlan0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
    link/ether b8:27:eb:6f:f8:8e brd ff:ff:ff:ff:ff:ff
    inet 10.3.141.1/24 brd 10.3.141.255 scope global noprefixroute wlan0
       valid_lft forever preferred_lft forever
    inet6 fe80::fa4b:2f7a:e92:4ac2/64 scope link
       valid_lft forever preferred_lft forever`

wlx74da38005318 
`4: wlx74da38005318: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group default qlen 1000
    link/ether 74:da:38:00:53:18 brd ff:ff:ff:ff:ff:ff
    inet 192.168.10.115/24 brd 192.168.10.255 scope global dynamic noprefixroute wlx74da38005318
       valid_lft 86320sec preferred_lft 75520sec
    inet6 fe80::dbcf:5157:dbba:d2de/64 scope link
       valid_lft forever preferred_lft forever`

**Comment by riker65** [comment]
do I need a gateway or route or bridge, I am not a network expert so.....working on it

**Comment by billz** [comment]
Search discussions for metrics or read [this post](https://www.reddit.com/r/RaspAP/comments/vl1g53/why_would_i_have_to_change_metrics/).
